packages/ERTP/src/issuer.js

```diff
@@ -0,0 +1,240 @@
+// Copyright (C) 2019 Agoric, under Apache License 2.0
+
+import harden from '@agoric/harden';
+import { assert, details } from '@agoric/assert';
+import makeStore from '@agoric/weak-store';
+import { E } from '@agoric/eventual-send';
+
+import makeAmountMath from './amountMath';
+
+function produceIssuer(allegedName, mathHelpersName = 'nat') {
+  assert(allegedName, details`allegedName must be truthy: ${allegedName}`);
```

Comment:
> you should assert that both allegedName and mathHelperName are strings.

I thought previously, we just wanted allegedName to be truthy. Since we have this check already, should be take it out and replace it with a string check?

The change to `makeAmountMath` to check typeof mathHelpersName is in.

Reply:
Now that you mention it, this is another one of those times where code that looked right (or even that I wrote) at one time looks bad to me at another time. Yes, I think you should replace the truthy check with an `assert.typeof(allegedName, 'string')` check.

Sorry for the thrashing, and doesn't need to be addressed in this PR. After all, the name already labels it "alleged".

Reply:
Done!